README.md

```diff
@@ -32,5 +25,11 @@ cd java-libkiwix
 AAR file will be generated in directory `lib/build/outputs/aar`
 
 
+# License
+
+[GPLv3](https://www.gnu.org/licenses/gpl-3.0) or later, see
+[COPYING](COPYING) for more details.
```

Comment:
There is no COPYING file.

Reply:
Thx, fixed with https://github.com/kiwix/java-libkiwix/pull/59